tools/control_board/generate_control_board.py

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env python2
```

Comment:
Use
```
#!/usr/bin/env python
```
python2 is not cross platform. 

even if pep-0394 mentioned that even brew have managed that yet and only arch user are the ones that use that.

Can you use that in a future but not now?